### ISSUE TITLE
Is there a series of these fonts without the word "System"

### ISSUE THREAD

**Issue by JonJonny** [wellsriley/YosemiteSanFranciscoFont#39]
Is there a series of these fonts without the word "System"

I love the font but if I put the "System San Francisco Display" or the "Text" series in my user font folder it causes a problem that I can explain if needed.

But I do like the font so do these fonts exist with other names 
Ex:
"San Francisco Bold" or something like that.

I talked to Apple about an issue and that issue was resolved by removing these fonts from the user folder either "System San Francisco Display WhatEver" or  "System San Francisco Text WhatEver"

Like I say I can explain the issue if requested but I just want to use the font for graphics etc.

**Comment by wellsriley** [comment]
TL;DR: you can't do that. It's in the README.

If you want the non-system fonts, you need to be a paid iOS Developer through ADC.

**Comment by JonJonny** [comment]
OK I know where to get them thank you
